src/ActionList/Item.tsx

```diff
@@ -75,9 +74,9 @@ export interface ItemProps extends React.ComponentPropsWithoutRef<'div'>, SxProp
 const getItemVariant = (variant = 'default', disabled?: boolean) => {
   if (disabled) {
     return {
-      color: themeGet('colors.text.disabled'),
-      iconColor: themeGet('colors.text.disabled'),
-      annotationColor: themeGet('colors.text.disabled'),
+      color: get('colors.text.disabled'),
```

Comment:
Is this the right `get` to use? I think we need `import {themeGet} from '@styled-system/theme-get'` or `import {themeGet} from '../constants'` in order to get proper theming, but I'm not certain.  @colebemis mentioned this on a PR the other day but I can't remember where.

Reply:
`get` calls `themeGet` internally, so theme values should be used:
https://github.com/primer/components/blob/7aeb53fec23a80414f6c1db8b31c06beb2073dd2/src/constants.ts#L8

Reply:
@smockle wouldn't we still need to import `get` here, or is it stored on `global` somewhere?